### ISSUE TITLE
[BUG]: How the hell do I get this to work, i've looked at the tutorial like 10,000 times, I still don't know what to do.

### ISSUE THREAD

**Issue by MilkyyTheGreat** [TheHunterManX/GBA-PK-multiplayer#5]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Description of the bug

Explain to me how to do this, like im 5 years old, please, I am not very intelligent.

### Steps To Reproduce

Change the IP of the Client side to the IP of my friends IP.

**Comment by CrushedBard971** [comment]
Make sure the ip was the correct one. The problem I was having was that I searched on google for my ip and it gave me the wrong one. What I should have done from the beginning was use the ipconfig command in the command prompt to get my real ip, and thats what worked for me. Not sure if this is your issue, but thought I'd share what got me stuck, hope it helps!

**Comment by fancythedeveloper** [comment]
Neither is this a bug as well

**Comment by ArtMatters** [comment]
ELI5:
1. Watch a video guide for port forwarding Minecraft.
2. Change the port from the video to the one for Pokemon.
3. Whoever does the port forwarding has to have the server file.
4. Whoever connects has to have the client file.
5. Change the IP in the client file to the visible IP of the host.
6. Make sure the host has a static IP.

**Comment by Catbear-22** [comment]
So im fairly experienced with setting up servers but I dont know which port im supposed to use for this to work. Usually I have a port number that Im told. I think its the number thats in the video that goes right after the IP. But I just want to make sure before I get fully invested into this. Thanks in advance

**Comment by BlazeITA** [comment]
> So im fairly experienced with setting up servers but I dont know which port im supposed to use for this to work. Usually I have a port number that Im told. I think its the number thats in the video that goes right after the IP. But I just want to make sure before I get fully invested into this. Thanks in advance
> 
> Edit: Nothing Im just dumb and needed to watch a little further in the video

Man, for sure im dumber than you, i watched the video like a thousand times but still cant understand how to setup a Radmin server and where to find the right port to give my mate.
Guys help me im getting crazy.

**Comment by Drakonas** [comment]
I've gotten this set up and We just get "timeout" errors on both sides. I think I'm chopping this up to be a very WIP alpha. Your success sadly is going to depend on how stable the connection is between the client and server. Sadly even if you are only a city away from your friend, the connection stability is at the whim of the internet... not much you can do until the developer improves the netcode.... which I'm sure they are already aware of... so no need to hammer them with this. It's alpha... Nothing is guaranteed yet.

**Comment by Drakonas** [comment]
Also, https://whatismyip.com will give you your IPv4 address which is what you want to use.

**Comment by televisionia** [comment]
You'd wanna use something like Hamachi and get your friend to join your network. The IP address is your IP in the network. To be honest I'm not really good at explaining stuff like this, so just look up youtube tutorials on how to use it. There are many for minecraft that would work with this mod if you change up the steps a bit. Just use the default port unless you know what you're doing. Personally, I used ZeroTier with my friend and it worked pretty fine: https://www.zerotier.com/

**Comment by TheHunterManX** [comment]
Port forward 4096, which is what the script uses, or simply use hamachi.
